Apps/WebClient/src/Server/Services/UserEmailService.cs

```diff
@@ -104,5 +120,16 @@ public bool UpdateUserEmail(string hdid, string email, Uri hostUri)
             this.logger.LogDebug($"Finished updating user email");
             return true;
         }
+
+        private async void UpdateNotificationSettings(UserProfile userProfile, string bearerToken)
+        {
+            // Update the notification settings
+            NotificationSettingsRequest request = new NotificationSettingsRequest(userProfile.Email, userProfile.PhoneNumber);
```

Comment:
I do like this better than object initialization... we should probably have some guidance on when object initialization should be used over constructors.

Reply:
well in the case of the constructor it carries logic to simplify the setting of the variables.